addons/web/static/src/webclient/actions/reports/layout_assets/layout_standard.scss

```diff
@@ -27,3 +27,9 @@
         margin-bottom: 0;
     }
 }
+
+.o_standard_header img {
+    max-height: 45px;
```

Comment:
You put a `max-height` of `45px` put the img already have a style of max-height 45px

Reply:
I don't understand could you clarify please